openedx/features/caliper_tracking/tests/expected/edx.librarycontentblock.content.assigned.json

```diff
@@ -0,0 +1,45 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Submitted",
+  "actor": {
+    "id": "http://localhost:18000/",
+    "type": "SoftwareApplication"
+  },
+  "eventTime": "2018-10-18T15:34:59.194Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.5",
+      "agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0",
+      "event_source": "server",
```

Comment:
`course_id` and `course_user_tags` are missing.

Reply:
done